docs/templates/tutorials/distributed-tuning.md

```diff
@@ -0,0 +1,122 @@
+# Distributed Tuning
+
+Keras Tuner makes it easy to perform distributed hyperparameter search. No changes to your code are needed to scale up from running single-threaded locally to running on dozens or hundreds of workers in parallel. Distributed Keras Tuner uses a chief-worker model. The chief runs a service to which the workers report results and query for the hyperparameters to try next. The chief should be run on a single-CPU worker (or alternatively as a separate process on one of the workers).
+
+### Configuring distributed mode
+
+Configuring distributed mode for Keras Tuner only requires setting three environment variables:
+
+**KERASTUNER_TUNER_ID**: This should be set to "chief" for the chief process. Other workers should be passed a unique ID (by convention, "tuner0", "tuner1", etc).
+
+**KERASTUNER_ORACLE_IP**: The IP address of the chief service.
+
+**KERASTUNER_ORACLE_PORT**: The port that the chief service should run on.
+
+The same code can be run on all workers. Additional considerations for distributed mode are:
+
+- All workers should have access to a centralized file system to which they can write their results.
+- All workers should be able to access the necessary training and validation data needed for tuning.
+- To support fault-tolerance, `overwrite` should be kept as `False` in `Tuner.__init__` (`False` is the default).
+
+Example chief setup (sample code for `run_tuning.py` at bottom of page):
+
+```bash
```

Comment:
The "bash" annotation will not be visible, please mention that these are bash scripts

Reply:
Done